### ISSUE TITLE
Wrong meetup date

### ISSUE THREAD

**Issue by saulecabrera** [montrealelixir/website#35]
If I'm not wrong, the meetup date presented should be the most recent one, in this case since there's no date for the an upcoming one it should show Wed, Jan 17, 2018.

<img width="853" alt="screen shot 2018-01-22 at 7 26 56 pm" src="https://user-images.githubusercontent.com/1423601/35252004-490ae4c8-ffac-11e7-9a22-70605febb1aa.png">

**Comment by nicholasjhenry** [comment]
@saulecabrera you're absolutely right. Possibly a caching issue?

**Comment by frahugo** [comment]
A fresh deploy to staging still shows the previous meetup.  This is most likely a bug.

**Comment by frahugo** [comment]
Confirmed.  I tried it locally and Meetup only returned one event: 

```
[
  %SocialFeeds.Meetup.Event{
    name: "Montreal Elixir Meetup",
    url: "https://www.meetup.com/montrealelixir/events/245950049/",
    utc_datetime: #DateTime<2018-01-17 23:30:00Z>,
    venue_address: "1100 avenue des Canadiens-de-Montréal, suite 150",
    venue_name: "SweetIQ"
  }
]
```

**Comment by frahugo** [comment]
This is clearly a bug in the Meetup API.  Following the documentation and trying their console here: https://secure.meetup.com/meetup_api/console/?path=/:urlname/events, it does not work.

The only work around is to specify a page size of 200 and get the last entry.

**Comment by saulecabrera** [comment]
@frahugo according to their documentation `page` is 200 by default, so maybe it can be omitted.

Another way of doing it which might be more declarative and would allow us to reduce the number of results might be:

```
page: 1,
scroll: next_upcoming,
desc: true
```
`next_upcoming` will get the next upcoming event and a filter of `upcoming | past` will be applied, so if there's no upcoming event the most recent past event will be given.

More details here https://www.meetup.com/meetup_api/docs/:urlname/events/#list

**Comment by frahugo** [comment]
Thanks @saulecabrera.  I didn't think of trying that scroll option.  It does work indeed, with the page size at 1 as well.  So this is clearly the fix.  Pull request submitted.

**Comment by nicholasjhenry** [comment]
Resolved in #41
